quarkus/config-api/src/main/java/org/keycloak/config/FeatureOptions.java

```diff
@@ -6,29 +6,35 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class FeatureOptions {
 
     public static final Option<List> FEATURES = new OptionBuilder("features", List.class, Profile.Feature.class)
             .category(OptionCategory.FEATURE)
-            .description("Enables a set of one or more features.")
-            .expectedValues(FeatureOptions::getFeatureValues)
+            .description("Enables a set of one or more features: " + getFeatureValues(true).stream().collect(Collectors.joining(",")))
```

Comment:
That's the only alternative we discussed.  Do you have another idea?  Otherwise we'll need to let it show all options: feature,feature:v1,feature:v2,...

Reply:
I wonder if we can't really use the `expectedValues` even with the custom format `feature[:v#]`. For features, we're now basically overriding the validation for expected values, so from validation perspective it does not matter what we have in `expectedValues`.